experimental/ir/zero_test.go

```diff
@@ -0,0 +1,123 @@
+package ir_test
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/bufbuild/protocompile/experimental/id"
+	"github.com/bufbuild/protocompile/experimental/ir"
+)
+
+func TestZero(t *testing.T) {
+	t.Parallel()
+
+	testZeroAny[*ir.File](t)
+	testZeroAny[ir.Import](t) // Import embeds *ir.File
+	testZeroAny[*ir.Imports](t)
```

Comment:
Related to the question below, this currently doesn't test `ir.Ref[T]` for `*ir.File` and `ir.Import`. If we don't need to enforce the `IsZero` interface, and these are valid values for `ir.Ref`, then I can simplify this.